spec/routing/comments_routing_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+RSpec.describe CommentsController, type: :routing do
+  describe "routing" do
+
+    it "routes to #index" do
+      expect(:get => "/comments").to route_to("comments#index")
+    end
+
+    it "routes to #new" do
+      expect(:get => "/comments/new").to route_to("comments#new")
+    end
+
+    it "routes to #show" do
+      expect(:get => "/comments/1").to route_to("comments#show", :id => "1")
+    end
+
+    it "routes to #edit" do
+      expect(:get => "/comments/1/edit").to route_to("comments#edit", :id => "1")
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)